src/resolvers/archDockerfile.ts

```diff
@@ -10,7 +12,7 @@ import { Bundle, FileInfo, Resolver } from '../resolver';
 type ArchSpecificDockerfile = [string, FileInfo];
 
 export default class ArchDockerfileResolver implements Resolver {
-	public priority = 1;
+	public priority = 3;
```

Comment:
This priority thing feels a bit weird - it doesn't block this PR but it seems like it would make more sense for the user of this library to choose the priorities, eg what if they want to add extra resolvers that sit inbetween the hardcoded ones in here?